src/main/java/com/noroff/mefit/backup/BackUp.java

```diff
@@ -0,0 +1,83 @@
+package com.noroff.mefit.backup;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Calendar;
+import java.util.Map;
+
+public class BackUp {
+    public static void DumpBackup() {
+        Process p;
+        ProcessBuilder pb;
+        pb = new ProcessBuilder(
+                "C:\\Program Files\\PostgreSQL\\14\\bin\\pg_dump.exe",
+                "--host=localhost",
+                "--port=5432",
+                "--username=postgres",
+                "--no-password",
+                "--format=custom",
+                "--blobs",
+                "--verbose", "--file", "C:\\Users\\OmaAli\\{data_source}-{timestamp}-dump.sql", "postgres");
+        try {
+            final Map<String, String> env = pb.environment();
+            env.put("PGPASSWORD", "admin");
```

Comment:
Does it work to use the Dockerfile env variables here? 😄

Reply:
Instead, yes